cluster.go

```diff
@@ -150,9 +151,18 @@ func NewCluster(
 	// a non permanent TTL.
 	c.peerManager.ImportPeersFromPeerstore(false, peerstore.AddressTTL)
 	// Attempt to connect to some peers (up to bootstrapCount)
-	actualCount := c.peerManager.Bootstrap(bootstrapCount)
-	// We cannot warn about this as this is normal if going to Join() later
-	logger.Debugf("bootstrap count %d", actualCount)
+	connectedPeers := c.peerManager.Bootstrap(bootstrapCount)
+	// We cannot warn when count is low as this as this is normal if going
+	// to Join() later.
+	logger.Debugf("bootstrap count %d", len(connectedPeers))
+	// Log a ping metric for every connected peer. This will make them
+	// visible as peers without having to wait for them to send one.
+	for _, p := range connectedPeers {
+		if err := c.logPingMetric(ctx, p); err != nil {
```

Comment:
peers we connected to will appear in the `current cluster peers` log message when cluster is Ready().